app/src/main/kotlin/social/plasma/ui/feed/FeedViewModel.kt

```diff
@@ -28,19 +26,17 @@ class FeedViewModel @Inject constructor(
 
     // TODO only subscribe to relays once during the entire app cycle
     // https://github.com/nostr-protocol/nips/blob/master/01.md#other-notes
-    private val relaySubscription =
-        relays.subscribe("wss://relay.damus.io").filter { it.contains("kind\":1") }
+    private val relaySubscription: Flow<Note> = runBlocking {
```

Comment:
Is this `runBlocking` appropriate, @jmateoac ?

Reply:
It blocks the UI thread which can lead to ANRs
Just pushed a commit that splits the subscription and does it on the IO thread pool